crates/optimism/evm/src/execute.rs

```diff
@@ -158,12 +158,12 @@ where
                     transaction_gas_limit: transaction.gas_limit(),
                     block_available_gas,
                 }
-                .into())
+                .into());
```

Comment:
```suggestion
                .into())
```
pls revert all these unrelated formatting changes. don't run `cargo fmt` for reth. use the nightly toolchain `cargo +nightly fmt`.

Reply:
@emhane, I have attempted to revert the formatting changes and used `cargo +nightly fmt` as you suggested, but I'm still facing issues. Could you please provide further guidance on how to resolve this?